parts/kubernetesmastercustomdata.yml

```diff
@@ -8,7 +8,7 @@ packages:
  - traceroute
 
 runcmd:
-- /bin/echo DAEMON_ARGS=--advertise-client-urls "http://127.0.0.1:2379,http://{{{masterPrivateIp}}}:2379" --listen-client-urls "http://0.0.0.0:2379,http://0.0.0.0:4001" | tee -a /etc/default/etcd
+- /bin/bash -c "/bin/echo DAEMON_ARGS=--name ""<<<variables('masterVMNames')[copyIndex()]>>>"" --initial-advertise-peer-urls ""<<<variables('masterEtcdPeerURLs')[copyIndex()]>>>"" --listen-peer-urls ""<<<variables('masterEtcdPeerURLs')[copyIndex()]>>>"" --advertise-client-urls ""<<<variables('masterEtcdClientURLs')[copyIndex()]>>>"" --listen-client-urls ""<<<concat(variables('masterEtcdClientURLs')[copyIndex()], ',http://127.0.0.1:', variables('masterEtcdClientPort'))>>>"" --initial-cluster-token ""k8s-etcd-cluster"" --initial-cluster ""<<<variables('masterEtcdClusterStates')[div(variables('masterCount'), 2)]>>>"" --initial-cluster-state ""new"" | tee -a /etc/default/etcd"
```

Comment:
Where did we end up on the whole double quote thing here? At the very least, I'd prefer they were all removed if they end up removing in the final file. Even better would be escaping them normally:

```
runcmd:
- /bin/bash -c "/bin/echo DAEMON_ARGS=--name \"<<<variables('masterVMNames')[copyIndex()]>>>\" --initial-advertise-peer-urls \"<<<variables('masterEtcdPeerURLs')[copyIndex()]>>>\" --listen-peer-urls \"<<<variables('masterEtcdPeerURLs')[copyIndex()]>>>\" --advertise-client-urls \"<<<variables('masterEtcdClientURLs')[copyIndex()]>>>\" --listen-client-urls \"<<<concat(variables('masterEtcdClientURLs')[copyIndex()], ',http://127.0.0.1:', variables('masterEtcdClientPort'))>>>\" --initial-cluster-token \"k8s-etcd-cluster\" --initial-cluster \"<<<variables('masterEtcdClusterStates')[div(variables('masterCount'), 2)]>>>\" --initial-cluster-state \"new\" | tee -a /etc/default/etcd"
```